apps/web/src/components/activity/ExecutionDetailsModal.tsx

```diff
@@ -44,7 +44,12 @@ export function ExecutionDetailsModal({
           paddingTop: '180px',
         },
       }}
-      title={<Title size={2}>Execution details</Title>}
+      title={
+        <>
+          <Title size={2}>Execution details</Title>
+          <small>( Updates every 3 sec )</small>
```

Comment:
Can we store the `refetchInterval` in a variable and then maybe use `date-fns` `formatDuration` utility to generate the output (https://date-fns.org/v2.14.0/docs/formatDuration)?